plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -210,7 +210,6 @@ public PluginCompatReport testPlugins()
         mconfig.userSettingsFile = config.getM2SettingsFile();
         // TODO REMOVE
         mconfig.userProperties.put( "failIfNoTests", "false" );
-        mconfig.userProperties.put( "argLine", "-XX:MaxPermSize=128m" );
```

Comment:
This clobbered both https://github.com/jenkinsci/plugin-pom/pull/131 and the corresponding local workaround in `~/.m2/settings.xml`, rendering PCT unusable with the current OpenJDK 8 on Ubuntu. Even before that, it was clobbering the default argline `-Xmx768M -Djava.awt.headless=true`.

Anyway I question the wisdom of blindly overriding JVM ergonomics during test runs. If there are indeed some test suites which for some reason require an outlandishly large permanent generation—and IIRC this is not even a thing in Java 8 anyway?—then they ought to be specifying that in their POMs.